config/mirrorbits.yaml

```diff
@@ -5,7 +5,6 @@ image:
   files:
     # repo & '@sha256' suffix here as we're retrieving a docker digest from updatecli
     repository: httpd@sha256
```

Comment:
```suggestion
```
The `repository` is already set to the same `httpd@sha256` value in the mirrorbits chart at https://github.com/jenkins-infra/helm-charts/blob/main/charts/mirrorbits/values.yaml#L12-L15, no need to set it again.